domains/mln.json

```diff
@@ -0,0 +1,10 @@
+{
+  "repo": "https://github.com/xmi1an",
+  "owner": {
+    "username": "xmi1an",
+    "email": "xnarcos300@gmail.com"
+  },
+  "record": {
+    "CNAME": "mln.github.io"
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.